doc/src/sgml/parallel.sgml

```diff
@@ -364,43 +555,69 @@ EXPLAIN SELECT * FROM pgbench_accounts WHERE filler LIKE '%x%';
     be performed while parallel query is in use, not even in the leader.
     When a query contains anything which is parallel unsafe, parallel query
     is completely disabled for that query.
+-->
+プランナは、クエリ中に実行される操作を<firstterm>パラレル安全（訳注： parallel safe）</>、<firstterm>パラレル制限（訳注：parallel restricted）</>、<firstterm>パラレル非安全（訳注：parallel unsafe）</>に分類します。
+パラレル制限操作は、パラレルクエリを利用中に、パラレルワーカー中では実行できないが、リーダーによって実行できる操作です。
+したがって、パラレル制限操作は、<literal>Gather</>ノードより下では決して実行されませんが、<literal>Gather</>ノードを含むプランの別の場所では実行されるかもしれません。
+パラレル非安全操作は、パラレルクエリ利用中に、リーダも含めて実行できない操作です。
+クエリがパラレル非安全なものを含む場合は、クエリ中でのパラレルクエリの利用は完全に禁止されます。
   </para>
 
   <para>
+<!--
     The following operations are always parallel restricted.
+-->
+次の操作は常にパラレル制限です。
   </para>
 
   <itemizedlist>
     <listitem>
       <para>
+<!--
         Scans of common table expressions (CTEs).
+-->
+共通テーブル式（CTE）のスキャン
       </para>
     </listitem>
 
     <listitem>
       <para>
+<!--
         Scans of temporary tables.
+-->
+一時テーブルのスキャン
       </para>
     </listitem>
 
     <listitem>
       <para>
+<!--
         Scans of foreign tables, unless the foreign data wrapper has
         an <literal>IsForeignScanParallelSafe</> API which indicates otherwise.
+-->
+外部テーブルのスキャン。
+外部データラッパが<literal>IsForeignScanParallelSafe</>パラレル安全を示しているAPIを持つ場合を除く。
```

Comment:
この部分は、外部テーブルのスキャンは原則としてパラレル制限ですよ、と言っていて、その中で、このAPIがotherwiseであることを示している場合は除きます、と言っているので、直感的に上記のようなコメントをしたのですが、このAPIはboolean型で、そんなに多弁ではないのですね。これはどう考えても誤解を招く表現なので、原文の修正を提案したいところです。
otherwiseとは「制限以外」ではなく「安全」のことであるのなら、「外部データラッパの～APIがパラレル安全を示す場合を除く」で良いと思います。「～APIを持ち」ですが、そのAPIが存在しなければ「APIがパラレル安全を示す」も成立しなくなるので、書かなくても誤解されないと思うのですが、どうですかね。